src/wizard/wizard-buttons.js

```diff
@@ -1,30 +1,31 @@
-function wizardButtonDirective (action) {
+(function () {
   'use strict';
-
-  angular.module('patternfly.wizard')
-    .directive(action, function () {
-      return {
-        restrict: 'A',
-        require: '^pf-wizard',
-        scope: {
-          callback: "=?"
-        },
-        link: function ($scope, $element, $attrs, wizard) {
-          $element.on("click", function (e) {
-            e.preventDefault();
-            $scope.$apply(function () {
-              // scope apply in button module
-              $scope.$eval($attrs[action]);
-              wizard[action.replace("pfWiz", "").toLowerCase()]($scope.callback);
+  function wizardButtonDirective (action) {
+    angular.module('patternfly.wizard')
+      .directive(action, function () {
+        return {
+          restrict: 'A',
+          require: '^pf-wizard',
+          scope: {
+            callback: "=?"
+          },
+          link: function ($scope, $element, $attrs, wizard) {
+            $element.on("click", function (e) {
+              e.preventDefault();
+              $scope.$apply(function () {
+                // scope apply in button module
+                $scope.$eval($attrs[action]);
+                wizard[action.replace("pfWiz", "").toLowerCase()]($scope.callback);
+              });
             });
-          });
-        }
-      };
-    });
-}
+          }
+        };
+      });
+  }
 
-wizardButtonDirective('pfWizNext');
-wizardButtonDirective('pfWizPrevious');
-wizardButtonDirective('pfWizFinish');
-wizardButtonDirective('pfWizCancel');
-wizardButtonDirective('pfWizReset');
+  wizardButtonDirective('pfWizNext');
+  wizardButtonDirective('pfWizPrevious');
+  wizardButtonDirective('pfWizFinish');
+  wizardButtonDirective('pfWizCancel');
+  wizardButtonDirective('pfWizReset');
+})();
```

Comment:
I would still like to see these function prefixed with pf. For example, pfWizardButtonDirective('pfWizNext')